deepchem/splits/splitters.py

```diff
@@ -893,6 +917,23 @@ class MolecularWeightSplitter(Splitter):
     Note
     ----
     This class requires RDKit to be installed.
+
+    Examples
+    --------
+
+    >>> import deepchem as dc
+    >>> import numpy as np
+    >>> # creation of demo data set with some smiles strings
+    >>> data_test= ["CC(C)Cl" , "CCC(C)CO" ,  "CCCCCCCO" , "CCCCCCCC(=O)OC" , "c3ccc2nc1ccccc1cc2c3" , "Nc2cccc3nc1ccccc1cc23" , "C1CCCCCC1" ]
```

Comment:
We can use a simple example of 5 data points, just carbon atoms `['C', 'CC', 'CCC', 'CCCC', 'CCCCC]`

Reply:
ok